index.html

```diff
@@ -462,13 +468,36 @@ <h4>The <dfn>ThingFilter</dfn> dictionary</h4>
             If <var>model</var> is not an object, throw <code>TypeError</code> and terminate these steps.
           </li>
           <li>
-            Let <var>instance</var> be the result of running the <a>Thing instantiation algorithm</a> with <var>model passed as parameter.
+            If <var>model</var> does not have an own property that is defined in <a>ThingFragment</a> with a default value, add that property and value to <var>model</var>.
           </li>
           <li>
-            Create an <a>ExposedThing</a> object <var>thing</var> initialized from <var>instance</var>.
+            Create an <a>ExposedThing</a> object <var>thing</var> initialized from <var>model</var>.
+          </li>
+          <li>
+            For each <a>ThingFragment</a> property, initialize the property based on the provided initial or default values provided to the local <a>WoT Runtime</a> implementation, for instance initialize:
+            <ol>
+              <li>
+                 the <code>id</code> property to be the final unique identifier of the <a>Thing</a>,
+              </li>
+              <li>
+                the <code>security</code> object of type <a>SecurityScheme</a> to represent the actual security scheme and its properties as set up by the implementation,
+              </li>
+              <li>
+                the <code>properties</code> property to be an array of <a>ThingProperty</a> objects in which the <code>read()</code> and <code>write()</code> methods are provided to define local methods to get and set the <a>Property</a> values,
```

Comment:
"**array** of <a>ThingProperty</a> objects"?
In the TD model, it is a map and most implementations would probably use a map for fast lookup. Yet this is an implementation detail.
I think we can skip this part and talk about adding the read/write methods similar to ConsumendThing, only that the access is local.
Initializing the state that holds the Property values should maybe go before adding the read and write?